crane/views/v2.py

```diff
@@ -0,0 +1,65 @@
+from flask import Blueprint, json, current_app, redirect
+from ..api import repository
+from .. import app_util
```

Comment:
We should use absolute imports instead of relative imports (i.e., from crane.api import repository).